src/styles/tokens.css

```
@@ -0,0 +1,140 @@
+   Dark-first, ANSI-accent, technical-not-costume-y.
+   Scheme toggled via [data-theme="dark"] / [data-theme="light"]
+   on <html>.
+   ============================================================ */
+
+@import url('https://fonts.googleapis.com/css2?family=Geist:wght@300;400;500;600;700&family=Geist+Mono:wght@400;500;600&display=swap');
+
+:root {
+  color-scheme: dark light;
+
+  /* --- Type families --- */
+  --font-sans: 'Geist', -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)